routes/web.php

```diff
@@ -14,5 +14,19 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
-});
+    return 'Home Page';
+})->name('home.index');
+
+// example route that we added as part of the tutorial
+Route::get('/contact', function() {
+    return 'Contact';
+})->name('home.contact');
+
+// example route to show the power of parameters (optional as well with ? + default)
+Route::get('/month/{id?}', function($id = '5') {
+    return 'Your month id is: ' . $id;
+})
+// ->where([
+//     'id' => '[0-9]+'
+// ])
```

Comment:
This is where you can define Route parameter patterns. (Route validations)
These ones are specific to the Route you chain them on though...